tutorials/evoked/plot_evoked_measures.py

```diff
@@ -0,0 +1,153 @@
+# the visual trials in each hemisphere. The :func:`mne.io.Evoked.get_peak`
+# method to achieve this. Here, we will focus on channels 'EEG 057' and
+# 'EEG 059' (left and right hemispheres, respectively).
+#
+# To mitigate high-frequency noise influencing peak measures, we can low-pass
+# filter the data with a cutoff of 20Hz or 30Hz. Here, we will do 20Hz. Note
+# that the Raw object of this sample dataset is lowpass filtered at 40Hz.
+
+# First we will xxtract the data from right trials in contralateral hemisphere
+
+# Define channel to use
+chan = 'EEG 054'
+
+# Copy and filter evoked
+right_vis_evoked = evokeds['visual/right'].copy()
+right_vis_evoked.filter(None, 20)
+right_vis_evoked.pick(chan)
+
+# Get peak latency and amplitude at EEG 054
+_, lh_lat, lh_amp = right_vis_evoked.get_peak(tmin=.07, tmax=.13,
+                                              mode='pos',
+                                              return_amplitude=True)
+
+# Convert amplitude from volts to microvolts
+lh_microvolts = lh_amp * 1e6
+
+# Plot the evoked trace with a point for the peak
+fig, ax = plt.subplots(nrows=1, ncols=1)
+fig = right_vis_evoked.plot(picks=chan, axes=ax,
+                            titles=f'Visual/Right Peak at {chan}',
+                            proj=True)
+ax.plot(lh_lat, lh_microvolts, 'ro')
+fig
+
+# Next, we will do the same for the left visual trials in
+# the right hemisphere
+chan = 'EEG 059'
+left_vis_evoked = evokeds['visual/right'].copy()
+left_vis_evoked.filter(None, 20)
+left_vis_evoked.pick(chan)
+_, rh_latency, rh_amplitude = left_vis_evoked.get_peak(tmin=.07, tmax=.13,
+                                                       mode='pos',
+                                                       return_amplitude=True)
+
+# Convert amplitude from volts to microvolts
+rh_microvolts = rh_amplitude * 1e6
+
+# Plot the evoked trace with a point for the peak
+fig, ax = plt.subplots(nrows=1, ncols=1)
+fig = right_vis_evoked.plot(picks=chan, axes=ax,
+                            titles=f'Visual/Left Peak at {chan}',
+                            proj=True)
+ax.plot(rh_latency, rh_microvolts, 'ro')
+fig
+
+
+###############################################################################
+# Obtaining Mean Amplitude in a Specified Time Window
+# ---------------------------------------------------------------
+#
+# Another common practice in ERP studies is to define a component (or effect)
+# as the mean amplitude within a specified time window. Sometimes, this also
+# involves extracting the mean over both time and space (sensors). The
+# following demonstrates how to do both. We will focus again on visual trials
+# in posterior electrodes.
+#
+# Below, we extract the mean amplitude from .08s to .12s for visual trials.
+# Given how the average amplitude over a time window is extracted, there is no
+# need to apply any additional lowpass filtering to the evoked data.
```

Comment:
I don't understand this comment, could you elaborate?

Reply:
I tried to elaborate a bit in the new commit. Briefly, averaging over a time window is a sort of low-pass filter, and reduces high-frequency noise in the signal. Hopefully the rewording helps. I tried to focus more now on just how to do it versus theory and rationale, which is more worthy of a discussion page.